azafea/tests/integration/__init__.py

```diff
@@ -119,5 +118,7 @@ def setup_teardown(self, request):
         self.clear_queues()
         self.ensure_no_queues()
 
+        self.db._engine.dispose()
```

Comment:
We discussed about that, it’s strange to get this error now. If the tests are not slower because of that, I think that it’s harmless.

Reply:
We had no errors before because we never reached the maximum number of connections allowed by postgres. But now that we have the v3 tests, we exceed this number and this causes connection errors to postgres. If I'm not mistaken with this line, at the end of each test we close the connection, which was not done before.